.github/workflows/ci.yml

```diff
@@ -651,7 +651,7 @@ jobs:
 
           cargo xtask test --llvm-cov
 
-      - name: check naga snapshots
+      - name: check Naga snapshots
```

Comment:
It just looks wrong :)

Reply:
I think the casing could remain, if we wanted to use backticks! 😀

Otherwise, I disagree; Naga is a proper noun, and thus should be capitalized.